### ISSUE TITLE
ESP32 download code

### ISSUE THREAD

**Issue by Montilien** [wuxx/Look4Sat-AntRunner-Controller#1]
Hi,
Congratulations for your very nice work.
So I am familiar to load code to ESP32 using Arduino IDE but I not with ESP tools.
Please, can you write a step by step code downloading tutorial please?
An other problem for me, on module picture I can see 2 ESP32's modules. I don't understand why two esp32 are necessary to the rotator work?
Thanks for your response
Regards - F1FHK

**Comment by Cyfarw9dd** [comment]
Hi,
This is BG2FZM, about load code using Arduino IDE. I'm afraid that this repository can only can run and flash on the esp-idf, which is a framework designed for the ESP chips for IOT functions, also more than that.
You can checkout this [tutorial](https://docs.espressif.com/projects/esp-idf/en/v5.3/esp32/get-started/index.html#installation) for further information. I recommend that you can use the vscode extension for the esp-idf and developing, which is more friendly and useful.
About the two chips, one is use for create a AP to let the other one and the mobile phone to check in, the other one running a tcp server to recviving the data from the Look4sat.
Hope it is useful
BA7KTD

**Comment by wuxx** [comment]
Yes, in fact, one ESP chip can do all the work, but I don't want to modify the source code in GRBL, because the workload of modification may be relatively large, so this is a lazy solution，may also be more flexible, because you can modify the source code yourself to adapt to other host software, and the source code will be simpler.

**Comment by Cyfarw9dd** [comment]
maybe I will try，I've seen a stepper motor example using with DRV8825, providing by the esp idf offical example code. I currently try to work out this figure, still need more time XD.

> Yes, in fact, one ESP chip can do all the work, but I don't want to modify the source code in GRBL, because the workload of modification may be relatively large, so this is a lazy solution，may also be more flexible, because you can modify the source code yourself to adapt to other host software, and the source code will be simpler.
